docs/api/covidcast-signals/covid-act-now.md

```diff
@@ -0,0 +1,116 @@
+---
+title: COVID Act Now
+parent: Data Sources and Signals
+grand_parent: COVIDcast Epidata API
+---
+
+# COVID Act Now (CAN)
+{: .no_toc}
+
+* **Source name:** `covid-act-now`
```

Comment:
FYI the data folder on staging has underscores instead of dashes

Reply:
Fixed in https://github.com/cmu-delphi/covidcast-indicators/pull/916